src/proxify.js

```diff
@@ -0,0 +1,144 @@
+/*
+Helper that can throw error when get/set undeclared properties.
+
+```javascript
+const obj = proxify({ foo: 42 });
+
+console.log(obj.foo);
+console.log(obj.bar); // throw
+```
+
+Some configuration:
+
+```javascript
+proxify({ foo: 42 }, {
+  name: 'object',
+  seal: true,
+  sealGet: false,
+  deprecated: [],
+  exposeApi: false,
+  apiNamespace: '__proxy__',
+});
+```
+
+- `name` : the name of the proxified object for warnings
+- `seal`: does `proxify` throw error on set undeclared properties?
+- `sealGet`: does `proxify` throw error on get undeclared properties? this options is separated from `seal` because getting undefined properties can be usefull for type checking for example
```

Comment:
```suggestion
- `sealGet`: does `proxify` throws an error when getting undeclared properties? This options is separated from `seal` because getting undefined properties can be useful for type checking for example
```